cypress/integration/errorPage.js

```diff
@@ -0,0 +1,31 @@
+import { getElement, renderedTitle } from '../support/bodyTestHelper';
+import { testResponseCode } from '../support/metaTestHelper';
+
+describe('Article Body Tests', () => {
+  // eslint-disable-next-line no-undef
+  before(() => {
+    cy.visit(`/news/articles/c9rpqyepmypo`, { failOnStatusCode: false });
+  });
+
+  it('should return a 404 error code', () => {
+    testResponseCode(`/news/articles/c9rpqyepmypo`, 404);
+  });
+
+  it('should display a relevant error message on screen', () => {
+    cy.visit(`/news/articles/c9rpqyepmypo`, { failOnStatusCode: false });
+    getElement('h1 span').should('contain', '404');
+    getElement('h1').should('contain', 'Page cannot be found');
```

Comment:
It'll be good to also pull out the text `Page cannot be found` into the config, since we'll soon have error pages in multiple languages.

Reply:
Just to document: This has been discussed, & will be done in a follow-up ticket.